### ISSUE TITLE
Stand output

### ISSUE THREAD

**Issue by simonbrd** [dpryan79/MethylDackel#133]
Hello is it possible with your tool to get the strand (+/-) in the bedgraph output file?

Thanks 
Simon

**Comment by dpryan79** [comment]
No, bedGraph has no strand column. If you explicitly need strand information you'll need a different format output (maybe one of the others produces that, I don't recall).
